File: DemoObjective-C/Component/MenuTableViewCell.m
//
//  MenuTableViewCell.m
//  DemoObjective-C
//
//  Created by Nguyễn Việt on 11/3/18.
//  Copyright © 2018 Quynhbkhn. All rights reserved.
//

#import "MenuTableViewCell.h"

@implementation MenuTableViewCell{
    NSArray *menuLb;
  //  NSArray *menuImage;
}



- (void)awakeFromNib {
    [super awakeFromNib];
    menuLb = @[@"Thong tin ca nhan", @"Crash App", @"Gioi thieu", @"Thoat"];
    
    // Initialization code
}


- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: Demo/component/MenuTableViewCell.swift
//
//  MenuTableViewCell.swift
//  Demo
//
//  Created by Nguyễn Việt on 10/31/18.
//  Copyright © 2018 Quynhbkhn. All rights reserved.
//

import UIKit

class MenuTableViewCell: UITableViewCell {
    
    @IBOutlet weak var mImage: UIImageView!
    
    @IBOutlet weak var mLabel: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }
    
    func setData(image: String, label: String){
        mImage.image = UIImage(named: image)
        mLabel.text = label
    }

    override func setSelected(_ selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
    
}
